<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SSS-PC" ignored="0" name="chromeTest" tests="3" failures="3" timestamp="24 Jul 2018 03:21:17 GMT" time="10.477" errors="0">
  <testcase name="FlipKart_TC1" time="8.751" classname="com.flipkart.testScripts.MultiBrowserTestCase2">
    <failure type="com.flipkart.utilities.MyOwnException" message="Login SUCCESSFUL
UNABLE TO INITIATE THE login PROCESS 
null value in entry: url=null
">
      <![CDATA[com.flipkart.utilities.MyOwnException: Login SUCCESSFUL
UNABLE TO INITIATE THE login PROCESS 
null value in entry: url=null

at com.flipkart.accelarators.Base.throwException(Base.java:208)
at com.flipkart.testScripts.MultiBrowserTestCase2.FlipKart_TC1(MultiBrowserTestCase2.java:42)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- FlipKart_TC1 -->
  <testcase name="FlipKart_TC1" time="8.565" classname="com.flipkart.testScripts.MultiBrowserTestCase3">
    <failure type="com.flipkart.utilities.MyOwnException" message="Login SUCCESSFUL
UNABLE TO INITIATE THE login PROCESS 
null value in entry: url=null
">
      <![CDATA[com.flipkart.utilities.MyOwnException: Login SUCCESSFUL
UNABLE TO INITIATE THE login PROCESS 
null value in entry: url=null

at com.flipkart.accelarators.Base.throwException(Base.java:208)
at com.flipkart.testScripts.MultiBrowserTestCase3.FlipKart_TC1(MultiBrowserTestCase3.java:42)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- FlipKart_TC1 -->
  <testcase name="FlipKart_TC1" time="8.744" classname="com.flipkart.testScripts.MultiBrowserTestCase">
    <failure type="com.flipkart.utilities.MyOwnException" message="Login SUCCESSFUL
UNABLE TO INITIATE THE login PROCESS 
null value in entry: url=null
">
      <![CDATA[com.flipkart.utilities.MyOwnException: Login SUCCESSFUL
UNABLE TO INITIATE THE login PROCESS 
null value in entry: url=null

at com.flipkart.accelarators.Base.throwException(Base.java:208)
at com.flipkart.testScripts.MultiBrowserTestCase.FlipKart_TC1(MultiBrowserTestCase.java:42)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- FlipKart_TC1 -->
</testsuite> <!-- chromeTest -->
